src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
+ * and returns an object where
+ * CSS properties are keys
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  let arr = sourceString.split(/[:;\n]/);
+
+  arr = arr.filter(item => item && item !== '  ');
+
+  for (let i = 0; i < arr.length; i = i + 2) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 5th item, you have iteration methods to do this